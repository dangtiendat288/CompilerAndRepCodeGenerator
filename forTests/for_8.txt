i, j, k, result;
{
    input i; 
    input j;
    input k;
    result = 0;
        FOR ( j = 0; j < 2; j = j + 1; )
        {
            result = i * j;
            output result;
        }
        output k;
        FOR ( k = 0; k < 2; k = k + 1; )
        {
            result = i * k;
            output result;
        }
}
5 3 7 8 