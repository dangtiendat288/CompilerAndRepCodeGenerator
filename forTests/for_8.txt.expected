0 5 14 0 5